rules/investigate-your-question-for-two-minutes-before-asking-someone-on-im/rule.md

```diff
@@ -1,28 +1,24 @@
 ---
 type: rule
-archivedreason: 
-title: Interruptions - Do you investigate your question for 2 minutes before asking someone on IM?
-guid: a876466b-f24d-4393-840d-cbd12946121e
+title: Interruptions - Do you investigate your question for 2 minutes before
+  asking someone on IM?
 uri: investigate-your-question-for-two-minutes-before-asking-someone-on-im
-created: 2015-10-05T05:42:39.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects:
-- interruptions-do-you-investigate-your-question-for-2-minutes-before-asking-someone-on-im
-
+  - interruptions-do-you-investigate-your-question-for-2-minutes-before-asking-someone-on-im
+created: 2015-10-05T05:42:39.000Z
+archivedreason: null
+guid: a876466b-f24d-4393-840d-cbd12946121e
 ---
-
 Always do a quick investigation before asking for help. When asking a question using IM, always start your conversation with "I have googled this and no luck".
 
-
-
-
-
 IM can interrupt a person when they are busy, so make sure that you are well prepared and have tried to find the solution another way first.
 
-
 <!--endintro-->
 
-![Figure: Don't make someone else have to Google something for you](lmgtfy.jpg)
+Its good practice to open your conversation with what investigation you have done. It will save the other person repeating work unnecessarily and is also a prompter for you to do the initial investigation.
```

Comment:
```suggestion
It's good practice to open your conversation with what investigation you have done. It will save the other person repeating work unnecessarily and is also a good prompt for you to do the initial investigation.
```